src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball one = new Lottery().getRundomBall();
+        Ball two = new Lottery().getRundomBall();
+        Ball three = new Lottery().getRundomBall();
```

Comment:
Don’t create several instances of the same class if you can use only one instance for your purposes
[common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-several-instances-of-the-same-class-if-you-can-use-only-one-instance-for-your-purposes)